app/api/admin_translations.py

```diff
@@ -0,0 +1,20 @@
+from flask import send_file, Blueprint
+import shutil
+import uuid
+import tempfile
+
+admin_blueprint = Blueprint('admin_blueprint', __name__, url_prefix='/admin/content/translations/all')
+temp_dir = tempfile.gettempdir()
+
+@admin_blueprint.route('/', methods=['GET'])
+def download_translations():
+    """Admin Translations Downloads"""
+    uuid_literal = uuid.uuid4()
+    zip_file = 'translations{}'
+    zip_filename = (zip_file + '.zip').format(uuid_literal)
+    shutil.make_archive(zip_file.format(uuid_literal), 'zip', temp_dir)
```

Comment:
.format(uuid_literal) is still duplicated

Reply:
Removed duplication